test/libsolidity/util/SoltestTypes.h

```diff
@@ -290,8 +290,6 @@ struct FunctionCall
 		LowLevel,
 		/// Marks a library deployment call.
 		Library,
-		/// Check that the storage of the current contract is empty or non-empty.
-		Storage,
 		/// Call to a builtin.
 		Builtin
```

Comment:
Should the builtins be documented somewhere? Now the `Storage` entry is gone, but it still exists under a new name, so it'd be nice if we can see somewhere what the builtins are

Reply:
Builtins get registered in `SemanticTest::initializeBuiltins()`, maybe I add here to refer to this function.